x-pack/plugins/reporting/server/routes/internal/management/jobs.ts

```diff
@@ -7,29 +7,26 @@
 
 import { schema } from '@kbn/config-schema';
 import { ROUTE_TAG_CAN_REDIRECT } from '@kbn/security-plugin/server';
-import { promisify } from 'util';
```

Comment:
Github shows this as a rename, so this diff makes a good outline of the goals of this PR. 

1. Use constants for the route path strings
2. Use helpers within route handler functions to avoid duplicated code